src/apprt/gtk/winproto/wayland.zig

```diff
@@ -3,12 +3,13 @@ const std = @import("std");
 const Allocator = std.mem.Allocator;
 const build_options = @import("build_options");
 
-const wayland = @import("wayland");
-const gtk = @import("gtk");
-const gtk4_layer_shell = @import("gtk4-layer-shell");
 const gdk = @import("gdk");
+const gdk_wayland = @import("gdk_wayland");
```

Comment:
If we're using gdk_{wayland,x11} modules then I think we can drop their headers in `c.zig`

Reply:
Fixed.